app/views/companies/jobs/show.html.erb

```diff
@@ -0,0 +1,29 @@
+<p><%= link_to "Back to #{@job.company.name}", company_path(@job.company), class:"btn btn-default" %> <%= link_to "Back to All Jobs", jobs_path, class:"btn btn-default" %></p>
+
+
+<h4>Title: <%= @job.title %></h4>
+
+Description: <%= @job.description %>
+<br>
+
+Level of Interest: <%= @job.level_of_interest %>
+<br>
+
+Company: <%= link_to @job.company.name, company_path(@job.company) %>
+
+Category: <%=@job.category.nil? ? 'not assigned' : @job.category.title%>
```

Comment:
we could extract this out to a method in your `Category` - it could either return the title or "not assigned"